tools/testsys/src/run.rs

```diff
@@ -68,6 +73,32 @@ pub(crate) struct Run {
 
     #[clap(flatten)]
     agent_images: TestsysImages,
+
+    // Migrations
+    /// Override the ami used for migrations. The ami will be pulled from ssm parameters for aws
+    /// variants if no override is provided.
+    #[clap(long, env = "TESTSYS_STARTING_IMAGE_ID")]
+    starting_image_id: Option<String>,
+
+    /// The starting version of bottlerocket migrations. This is required for all migrations tests.
```

Comment:
nit: I'd either remove these "bottlerocket" references or capitalize them

```suggestion
    /// The starting version for migrations. This is required for all migrations tests.
```